specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2023-08-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,2890 @@
+            "schema": {
+              "$ref": "#/definitions/Replica"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Replicas_Create": {
+            "$ref": "./examples/ConfigurationStoresCreateReplica.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      },
+      "delete": {
+        "tags": [
+          "Replicas"
+        ],
+        "description": "Deletes a replica.",
+        "operationId": "Replicas_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ConfigStoreNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "replicaName",
+            "in": "path",
+            "description": "The name of the replica.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly."
+          },
+          "202": {
+            "description": "The request was successful; the operation will complete asynchronously.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "No Content - the specified resource was not found."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Replicas_Delete": {
+            "$ref": "./examples/ConfigurationStoresDeleteReplica.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AppConfiguration/configurationStores/{configStoreName}/snapshots/{snapshotName}": {
+      "get": {
+        "tags": [
+          "Snapshots"
+        ],
```

Comment:
Not sure what do you meant by "Following KeyValue, with both being proxies for data plane resources.", why is Collection GET not implemented for Snapshots?

Reply:
It wasn't added for snapshots because we followed the modeling for the existing KeyValue proxy resource which doesn't support it.

Both are proxies for data plane resources and are for supporting ARM template deployments. Note: Other scenarios involving KeyValues/Snapshots are expected to use our data plane API instead.